services/workflows-service/src/workflow/hook-callback-handler.service.ts

```diff
@@ -9,6 +9,7 @@ import fs from 'fs';
 import { CustomerService } from '@/customer/customer.service';
 import { TProjectId, TProjectIds } from '@/types';
 import { TDocumentsWithoutPageType } from '@/common/types';
+import { fromBuffer } from 'file-type';
```

Comment:
Its a bit conusing when using it like that, consider changing to `import { fromBuffer as getFileTypeFromBuffer} from 'file-type';`